webapp/src/main/java/com/box/l10n/mojito/service/evolve/dto/CourseRequestDTO.java

```diff
@@ -0,0 +1,26 @@
+package com.box.l10n.mojito.service.evolve.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CourseRequestDTO {
+  private int id;
+
+  @JsonProperty("custom_j")
```

Comment:
Ah this might make it difficult to generalise the `custom_j` fields usage, don't spend too much time trying to generalise it if this becomes an issue